configs/testnet.json

```diff
@@ -65,23 +71,31 @@
   "quick_select_deposit_options": [
     {
       "label_denom": "swth",
-      "target_denom" : "swth"
+      "target_denom": "swth"
     },
     {
       "label_denom": "usdc",
-      "target_denom" : "usdc"
+      "target_denom": "usdc"
     },
     {
       "label_denom": "eth",
-      "target_denom" : "eth"
+      "target_denom": "eth"
     },
     {
       "label_denom": "dai",
-      "target_denom" : "dai"
+      "target_denom": "dai"
     }
   ],
+  "chain_fee_token_map": {
+    "Binance Smart Chain": {
+      "fee_denom": "brdg/a493620f7d356bb7fee3a5ea8511d70d5ac728232cc5fe4a25dbd8028993a0e7",
+      "decimals": 18
+    }
+  },
```

Comment:
Sorry, think Carbon Testnet was just reseeded. Could you change the following:
- `Binance Smart Chain` => `BSC testnet`
- `brdg/a493620f7d356bb7fee3a5ea8511d70d5ac728232cc5fe4a25dbd8028993a0e7` => `brdg/058df3a0734c28f905ea8a9ce7f8183a6f76fb10ed1b8cf70acb5f697f5d7009`